client/rest/src/main/java/org/elasticsearch/client/RestClient.java

```diff
@@ -663,15 +644,10 @@ public void onFailure(Exception exception) {
          */
```

Comment:
The javadoc on this is going to be wrong then.

Reply:
good point. fixing